apps/explorer/src/App.vue

```diff
@@ -22,9 +25,93 @@ export default Vue.extend({
   },
   //Add footer + nav declaration
   data() {
-    return {
-      pageName: '/'
+    return {}
+  },
+  created() {
+    this.$options.sockets.connect = () => {
+      if (!this.pageName || this.pageName === 'blocks' || this.pageName === 'transactions') {
+        this.setPastData()
+      } else {
+        setTimeout(() => {
+          this.setPastData()
+        }, 1000)
+      }
+    }
+  },
+  methods: {
+    setPastData() {
+      this.$socket.emit(
+        sEvents.pastTxs,
+        {
+          limit: 100,
+          page: 0
+        },
+        (err, txs) => {
+          this.$store.commit(sEvents.newTx, txs)
+          if (txs && txs.length > 0) {
+            this.$eventHub.$emit(sEvents.pastTxsR)
+            this.$eventHub.$emit(sEvents.newTx, new Tx(txs[0]))
+          }
+        }
+      )
+      this.$socket.emit(
+        sEvents.pastBlocks,
+        {
+          limit: 100,
+          page: 0
+        },
+        (err, blocks) => {
+          this.$store.commit(sEvents.newBlock, blocks)
+          if (blocks && blocks.length > 0) {
+            this.$eventHub.$emit(sEvents.newBlock, new Block(blocks[0]))
+            this.$eventHub.$emit(sEvents.pastBlocksR)
+          }
+        }
+      )
+      this.$socket.emit(
+        sEvents.pendingTxs,
+        {
+          limit: 100,
+          page: 0
+        },
+        (err, pTxs) => {
+          this.$store.commit(sEvents.newPendingTx, pTxs)
+          if (pTxs && pTxs.length > 0) {
+            this.$eventHub.$emit(sEvents.newPendingTx)
+          }
+        }
+      )
+
+      this.$socket.emit(
+        sEvents.getUncles,
+        {
+          limit: 100,
+          page: 0
+        },
+        (err, uncles) => {
+          this.$store.commit(sEvents.newUncle, uncles)
+          if (uncles && uncles.length > 0) {
+            this.$eventHub.$emit(sEvents.newUncle)
+          }
+        }
+      )
     }
   }
+
+  /*
```

Comment:
@olgakup if this section is not going to be needed better to remove this commented section